srcpkgs/avahi-discover/template

```diff
@@ -67,40 +78,66 @@ do_install() {
 	done
 	mv ${wrksrc}/tmpinstall/usr/include/avahi-g* ${DESTDIR}/usr/include
 	mv ${wrksrc}/tmpinstall/usr/include/avahi-ui* ${DESTDIR}/usr/include
-	mv ${wrksrc}/tmpinstall/usr/lib/python2.7 ${DESTDIR}/usr/lib
+	mv ${wrksrc}/tmpinstall/usr/include/avahi-qt* ${DESTDIR}/usr/include
+	mv ${wrksrc}/tmpinstall/usr/include/avahi-libevent* ${DESTDIR}/usr/include
+	mv ${wrksrc}/tmpinstall/usr/lib/python3.8 ${DESTDIR}/usr/lib
+	# Python files are compatible with both 2.7 and 3.x, make both subpackages
+	vmkdir ${py2_lib}
+	vcopy ${DESTDIR}/${py3_sitelib} ${py2_lib}
 	mv ${wrksrc}/tmpinstall/usr/lib/*avahi-g* ${DESTDIR}/usr/lib
 	mv ${wrksrc}/tmpinstall/usr/lib/*avahi-ui* ${DESTDIR}/usr/lib
+	mv ${wrksrc}/tmpinstall/usr/lib/*avahi-qt* ${DESTDIR}/usr/lib
+	mv ${wrksrc}/tmpinstall/usr/lib/*avahi-libevent* ${DESTDIR}/usr/lib
 	mv ${wrksrc}/tmpinstall/usr/lib/pkgconfig/avahi-g* ${DESTDIR}/usr/lib/pkgconfig
 	mv ${wrksrc}/tmpinstall/usr/lib/pkgconfig/avahi-ui* ${DESTDIR}/usr/lib/pkgconfig
+	mv ${wrksrc}/tmpinstall/usr/lib/pkgconfig/avahi-qt* ${DESTDIR}/usr/lib/pkgconfig
+	mv ${wrksrc}/tmpinstall/usr/lib/pkgconfig/avahi-libevent* ${DESTDIR}/usr/lib/pkgconfig
 	mv ${wrksrc}/tmpinstall/usr/share/avahi/interfaces/*.ui ${DESTDIR}/usr/share/avahi/interfaces
+
 	if [ "$build_option_gir" ]; then
 		mv ${wrksrc}/tmpinstall/usr/lib/girepository-1.0 ${DESTDIR}/usr/lib
 		mv ${wrksrc}/tmpinstall/usr/share/gir-1.0 ${DESTDIR}/usr/share
 	fi
 }
 
-avahi-python-utils_package() {
-	depends="avahi-python>=$version"
+avahi-python3-utils_package() {
+	depends="avahi-python3>=${version}_${revision}"
 	short_desc="Avahi python utilities"
 	archs=noarch
+	python_version=3
 	pkg_install() {
 		vmove usr/bin/avahi-bookmarks
 		vmove "usr/share/man/man1/avahi-bookmarks*"
 	}
 }
 
+avahi-python-utils_package() {
+	short_desc="Avahi python utilities - transitional dummy pkg"
+	depends="avahi-python3-utils>=${version}_${revision}"
+	archs=noarch
+	build_style=meta
+}
+
 avahi-python_package() {
 	depends="python-dbus"
 	short_desc="Python utility package for Avahi"
 	archs=noarch
-	pycompile_module="avahi"
 	pkg_install() {
-		vmove usr/lib/python2.7/site-packages/avahi
+		vmove ${py2_sitelib}/avahi
+	}
+}
+
+avahi-python3_package() {
+	depends="python3-dbus"
+	short_desc="Python3 utility package for Avahi"
+	archs=noarch
+	pkg_install() {
+		vmove ${py3_sitelib}/avahi
 	}
 }
 
 avahi-glib-libs-devel_package() {
-	depends="avahi-libs-devel>=${version}_${revision} avahi-glib-libs>=${version}_${revision}"
+	depends="avahi-libs-devel>=${version} avahi-glib-libs>=${version}_${revision}"
```

Comment:
why do you drop `_${revision}` from a lot of the dependencies?

Reply:
Yea really don't like that its a mix of `>=${version}` and ``>=${version}`_${revision}` all over the builds.

Reply:
Hm ok, not sure what the best approach is here. There is no requirement for a revision in those patterns so I guess its fine.

Reply:
This is one of those problems I was hoping to fix with the template unification, until you discovered the dependency cycle. I thought about dropping the version altogether (option 4), but keeping a version with no revision at least lets us catch version incompatibilities which should never happen. I will keep as-is.